pkg/config/config_test.go

```diff
@@ -283,7 +283,7 @@ func TestGetUpdateDataPeriodMock(t *testing.T) {
 		t.Fatalf("ParseConfiguration failed with err %v", err)
 	}
 
-	expected := time.Duration(30) * time.Second
+	expected := time.Duration(60) * time.Second
```

Comment:
```suggestion
	if cfg.GetUpdateDataPeriod() != 1*time.Minute {
```
```suggestion
	expected := time.Duration(1) * time.Minute
```
This should be the way of testing